src/main/java/juja/microservices/teams/entity/Team.java

```diff
@@ -32,14 +37,13 @@ public Team(@JsonProperty("from") String from,
                 @JsonProperty("uuidOne") String uuidOne,
                 @JsonProperty("uuidTwo") String uuidTwo,
                 @JsonProperty("uuidThree") String uuidThree,
-                @JsonProperty("uuidFour") String uuidFour,
-                @JsonProperty("startDate") Date startDate) {
+                @JsonProperty("uuidFour") String uuidFour) {
         this.from = from;
         this.uuidOne = uuidOne;
         this.uuidTwo = uuidTwo;
         this.uuidThree = uuidThree;
         this.uuidFour = uuidFour;
-        this.startDate = startDate;
+        this.startDate = new Date();
```

Comment:
It is not suitable constructor for our logic. Because every time we want to get instance of team from client side (e.g. web UI) we have to map that json to our entity Team. And every time the start date will reinstall. It could be the reason to create separate TeamRequest or come up a new solution.
The second thing is dismiss date. I think we have to set it in that moment when the start date is set. Plus 1 month. Later cron could check dismissed dates and run dismiss method automaticaly.